demo/config/services.yaml

```diff
@@ -19,10 +19,6 @@ services:
             - '../src/Entity/'
             - '../src/Kernel.php'
 
-    Symfony\AI\McpSdk\Capability\Tool\ToolExecutorInterface:
-        class: Symfony\AI\McpSdk\Capability\ToolChain
-        arguments:
-            - ['@App\MCP\Tools\CurrentTimeTool']
-
-    Symfony\AI\McpSdk\Capability\Tool\CollectionInterface:
-        alias: Symfony\AI\McpSdk\Capability\Tool\ToolExecutorInterface
+    App\MCP\Tools\CurrentTimeTool:
+        tags:
+            - { name: 'mcp_tool' }
```

Comment:
let's use `#[Autoconfigure]` instead - at least for now - doing autoconfiguration via interface would be the next step maybe